contracts/APRCalculator/modules/MacroFactor/MacroFactor.sol

```diff
@@ -93,9 +93,9 @@ abstract contract MacroFactor is IMacroFactor, Price {
      * @notice Calculate the Simple Moving Average (SMA) ratio and set the macro factor accordingly.
      */
     function _calcSMA() private view returns (uint256 smaRatio) {
-        uint256 smaFast = smaFastSum / FAST_SMA;
+        uint256 smaFast = (smaFastSum * DENOMINATOR) / FAST_SMA;
         uint256 smaSlow = smaSlowSum / SLOW_SMA;
```

Comment:
No, because the voted price cannot be 0, should always be 1, and the sum tasks the last number of voted prices which will always be = or > SLOW_SMA. We need to multiply the fastSum, so later we can divide by the slow without getting 0.